### PR TITLE
Add fixes for various anime due to weird name changes

### PR DESCRIPTION
**Pull request by adamus1red**
* Zetsuen no Tempest
  * Added "Blast of Tempest" & "Blast of Tempest: The Civilization Blaster" synonym
* Elfen Lied
  * Added "Erufen rîto" synonym
* Grimgar of Fantasy & Ash
  *  Added "Hai to gensô no Grimgar" Synonym
* One Punch Man
  * Added various synonyms
* Kawaii dake ja Nai Shikimori-san
  * Added various synonyms

### CODE REVIEW

File: series-tvdb.en.yaml [hunk 1]
```diff
@@ -1054,6 +1078,14 @@ entries:
         anilist-id: 20993
         start: 13
 
+  - title: "Kawaii dake ja Nai Shikimori-san"
```
Comment (reconman) [inline series-tvdb.en.yaml:1081]: The English title should be in `title:`, the Japanese one in synonyms.

File: series-tvdb.en.yaml [hunk 2]
```diff
@@ -1356,3 +1388,11 @@ entries:
         start: 7
       - season: 3
         anilist-id: 122292
+
+  - title: "Zetsuen no Tempest"
```
Comment (reconman) [inline series-tvdb.en.yaml:1392]: Same here

File: series-tvdb.en.yaml [hunk 3]
```diff
@@ -566,6 +574,13 @@ entries:
       - season: 3
         anilist-id: 11061
 
+  - title: "Hyakuren no Haou to Seiyaku no Valkyria"
```
Comment (reconman) [inline series-tvdb.en.yaml:577]: Same here

**Review by reconman — CHANGES_REQUESTED** [review]
Sorry for the late review, I somehow missed this PR.

File: series-tvdb.en.yaml (hunk 1 above)
Comment (adamus1red) [reply series-tvdb.en.yaml]: Aye, no problem.

**Pull request merged**